cpp/src/codegen/llvm_generator.cc

```diff
@@ -826,8 +922,8 @@ llvm::Value *LLVMGenerator::Visitor::GetBufferReference(int idx,
     slot_ref = generator_->GetDataReference(arg_addrs_, idx, field);
     break;
 
-  default:
```

Comment:
should we not leave this as is for future safety?

Reply:
Removing the default will cause a compile time error (enumeration value not handled in switch). I felt that's better than a runtime dcheck.

Reply:
ah nice. did not know that, agreed compile time is ideal.